Projects/App/Sources/UI/Organization/OrganizationList/OrganizationListViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  OrganizationListViewController.swift
+//  App
+//
+//  Created by 김태호 on 2022/10/16.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import Combine
+import SnapKit
+import DesignSystem
+
+final class OrganizationListViewController: UIViewController {
+    
+    // MARK: Properties
+    
+    private var viewModel = OrganizationListViewModel()
+    
+    private var subscriptionSet = Set<AnyCancellable>()
+    
+    private var orgNameArray: [String] = []
+    
+    private let titleLabel = UILabel()
+    private let searchingTextFieldView = ShadowTextFieldView()
+    private lazy var searchingTextField = searchingTextFieldView.textField
+    private let tableView = UITableView()
+    
+    // MARK: LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+        bindUI()
+        searchingTextField.delegate = self
+    }
+    
+    // MARK: Function
+    
+}
+
+extension OrganizationListViewController {
+    
+    // MARK: UI Function
+    
+    private func configureUI() {
+        view.backgroundColor = .white
+        
+        titleLabel.text = "참여할 대학교를 알려주세요"
+        // TODO: system font size로 바꾸기
+        titleLabel.font = UIFont.boldSystemFont(ofSize: 26)
+        
+        tableView.dataSource = self
+        tableView.register(OrganizationListCell.self, forCellReuseIdentifier: OrganizationListCell.identifier)
+    }
+    
+    private func createLayout() {
+        view.addSubviews([titleLabel, searchingTextFieldView, tableView])
+        
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(16)
+            make.left.equalTo(view.snp.left).offset(20)
+        }
+        
+        searchingTextFieldView.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).offset(16)
+            make.left.equalTo(view.snp.left).offset(20)
+            make.right.equalTo(view.snp.right).offset(-20)
+            make.height.equalTo(45)
+        }
+        
+        tableView.snp.makeConstraints { make in
+            make.top.equalTo(searchingTextFieldView.snp.bottom).offset(16)
+            make.left.equalToSuperview()
+            make.right.equalToSuperview()
+            make.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom)
+        }
+    }
+    
+    private func bindUI() {
+        self.viewModel.$searchingArray.sink { orgNameArray in
+            self.orgNameArray = orgNameArray
+            self.tableView.reloadData()
+        }.store(in: &subscriptionSet)
```

Comment:
p3
가독성을 위해 바꾸면 좋을 것 같아요
```suggestion
        }
        .store(in: &subscriptionSet)
```

Reply:
130dc01 반영했습니다